### DISCUSSION TITLE
Polyphony problem

### DISCUSSION THREAD

**Discussion by biamau962** [ElectroTechnique/TSynth-Teensy4.1#135] [T4.1 code development]
Sorry for my bad English
I play any note and I hold down the corrisponding key then i play twelve notes (equivalent to the polyphony of TSYNTH) one or two at the time, once the twelft has been reached the note plays and when held down it goes out. This only happens if the note held down corresponds to voice number 1 or 2 (first and second squares on the dislay).

**Comment by Electrotechnique** [comment]
Yes, I can confirm this.

**Reply by biamau962** [reply]
thank you very much, it works perfectly
